CONTRIBUTING.md

```diff
@@ -54,14 +54,15 @@ Following the targets that can be used to test your changes locally.
 | make lint |  Check the code implementation | yes   |
 | make test-coverage |  Run coveralls to check the % of code covered by tests | yes   |
 | make check-testdata |  Checks if the testdata dir is updated with the latest changes | yes   |
+| make test-e2e-local |  Runs the CI e2e tests locally | no   |
```

Comment:
Could you please revert the changes that has no relation with the scope of this pr? (E.g the spaces an just keep what is related to the new target)